autoreject/tests/test_autoreject.py

```diff
@@ -318,3 +319,33 @@ def test_io():
     assert_array_equal(epochs_clean1.get_data(), epochs_clean3.get_data())
     assert_array_equal(reject_log1.labels, reject_log2.labels)
     assert_array_equal(reject_log1.labels, reject_log3.labels)
+
+
+def test_fnirs():
+    # Test that autoreject runs on fNIRS data.
```

Comment:
```suggestion
    """Test that autoreject runs on fNIRS data."""
```

Just a proper docstring so pytest can use it